### ISSUE TITLE
[Suggestion]: Pets in bowels

### ISSUE THREAD

**Issue by Flaming-skeleton-guitar** [nonamecrackers2/crackers-wither-storm-mod#1392]
### Desription of feature request

Make it so pets or ridden horses/pigs follow you into the bowels, similarly to Reuben in mcsm
Could help with the command block fight

**Comment by TaintedBlockmObile** [comment]
How would this work

**Comment by Nov1st2k10** [comment]
No, they aren't going to add MCSM comtent, however if you are interested in MCSM content you should look at the "Story Mod" mod

**Comment by Flaming-skeleton-guitar** [comment]
> How would this work

Maybe they teleport into the dimension with you

**Comment by mattdafilipino** [comment]
Addon to this goofy idea: Reuben dies at the end of the final battle no matter what and after u defeat the witherstorm you see a random ass pig laying in the ground and you have a whole skyrim dialogue talk with him until he turns into a porkchop

**Comment by Flaming-skeleton-guitar** [comment]
Misclick

**Comment by Flaming-skeleton-guitar** [comment]
> No, they aren't going to add MCSM comtent, however if you are interested in MCSM content you should look at the "Story Mod" mod

I don't mean adding reuben, I mean teleporting pets into the bowels

**Comment by Pandaboi25x** [comment]
> > No, they aren't going to add MCSM comtent, however if you are interested in MCSM content you should look at the "Story Mod" mod
> 
> I don't mean adding reuben, I mean teleporting pets into the bowels

reuben is getting added what they mean is prob not making the mobs go to the bowels
